Assets/Scripts/Enemies/PufferbombSpawner.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PufferbombSpawner : MonoBehaviour
+{
+    public float height; //how far up to float Pufferbombs from initial y position
+    public float speed; //speed to float Pufferbombs
+
+    private bool floatUp;
+    private float finalHeight;
+
+    void Start()
+    {
+        //disable Pufferbomb script for all Pufferbombs
+        foreach (Transform child in transform)
+        {
+            if (child.GetComponent<Pufferbomb>() != null)
+            {
+                child.GetComponent<Pufferbomb>().enabled = true;
```

Comment:
I'm a little confused here. At first I read that you disabled them until the trigger was activated but then you activated here? Looking into it further I am unsure if this is needed. Please justify why this is needed and if not then please remove

Reply:
I think I might have copied and pasted without changing it to false, so I fixed it.